frontend/src/features/mapSideBar/MotiInventoryContainer.tsx

```diff
@@ -62,6 +68,24 @@ export const MotiInventoryContainer: React.FunctionComponent<IMotiInventoryConta
     func();
   }, [getPropertyWithPid, isMounted, props.pid]);
 
+  const {
+    getPropertyAssociations,
+    isLoading: propertyAssociationsLoading,
+  } = usePropertyAssociations();
+
+  useEffect(() => {
+    const func = async () => {
+      if (props.pid !== undefined) {
+        const response = await getPropertyAssociations(props.pid);
+        if (response?.id !== undefined) {
+          setPropertyAssociations(response);
```

Comment:
The useApiWrapper also returns the response, and does so in a way that respects whether or not the component is mounted. I think this may bypass that check.

Reply:
Sure thing! I was following the same retrieval as the other api calls

Reply:
Yeah, I think that was my fault, I'll need to log tech debt to go fix those.